lighthouse/src/main.rs

```diff
@@ -410,7 +417,7 @@ fn main() {
     let is_beacon_node = matches.subcommand_name() == Some("beacon_node");
     if is_beacon_node && !matches.get_flag(DISABLE_MALLOC_TUNING_FLAG) {
         if let Err(e) = configure_memory_allocator() {
-            eprintln!(
+            error!(
```

Comment:
since we're probably going to move the tracing subscriber further downstream we'll need to keep the `eprintln`